src/GafferBindings/CompoundNumericPlugBinding.cpp

```diff
@@ -127,6 +182,15 @@ void bind()
 {
 	typedef typename T::ValueType V;
 
+	enum_<IECore::GeometricData::Interpretation>("Interpretation")
```

Comment:
Yes, that's in there because had my own enum for a bit and then switched to the IECore one. Search/Replace "fixed" this one as well, but you're right - I need to get rid of it.